connectors/src/connectors/slack/temporal/activities.ts

```diff
@@ -57,9 +62,9 @@ export async function getChannels(
     if (c.error) {
       throw new Error(c.error);
     }
-    if (!c.channels) {
+    if (c.channels === undefined) {
```

Comment:
OOC, can `c.channels` be null ?

Reply:
No it can't. Thanks for checking. This is actually a crucial part of the code. If we were to return an empty list by mistake, we would delete all documents from the data source.